package.json

```diff
@@ -12,8 +12,9 @@
     "build": "ember build",
     "preversion": "git clean -df && npm test",
     "start": "ember server",
-    "test": "ember try:testall",
-    "docs": "node ./docs"
+    "test": "ember try:testall && mocha tests/**/*-test-node.js",
+    "docs": "node ./docs",
+    "mocha": "mocha tests/**/*-test-node.js"
```

Comment:
Mocha is added below in `devDependencies`.  `scripts` are ran with the `node_modules/.bin` directory in `$PATH`, so all binaries added by deps/devDeps are available.